app/src/main/kotlin/com/antosik/benchproject/app/main/view/application/MyApplication.kt

```diff
@@ -1,4 +1,4 @@
-package com.antosik.benchproject
+package com.antosik.benchproject.app.main.view.application
```

Comment:
`view` package is redundant, actually `Application` is not a `View` :D

Reply:
Done 😄